uboot-rockchip: use u-boot-rockchip.bin

RadxaNaoki opened the pull request:
use u-boot-rockchip.bin to copy SPL/TPL/U-Boot to the image.

since binman was used in mainline u-boot for rockchip, we can use u-boot-rockchip.bin instead of idbloader.img and u-boot.itb.

1715173329 reviewed (approved):
`Reviewed-by: Tianling Shen <cnsztl@immortalwrt.org>`

Thank you!

1715173329 commented on target/linux/rockchip/image/Makefile:
```diff
@@ -29,7 +29,7 @@ define Build/pine64-img
 
 	# Generate a new partition table in $@ with 32 MiB of 
-	# alignment padding for the idbloader and u-boot to fit:
+	# alignment padding for the u-boot-rockchip.bin to fit:
```
```suggestion
	# alignment padding for the u-boot-rockchip.bin (idbloader + u-boot) to fit:
```
make it bit more clear

RadxaNaoki committed:
uboot-rockchip: use u-boot-rockchip.bin

use u-boot-rockchip.bin to copy SPL/TPL/U-Boot to the image.

since binman was used in mainline u-boot for rockchip, we can use
u-boot-rockchip.bin instead of idbloader.img and u-boot.itb.

Reviewed-by: Tianling Shen <cnsztl@immortalwrt.org>
Signed-off-by: FUKAUMI Naoki <naoki@radxa.com>
Link: https://github.com/openwrt/openwrt/pull/15815
Signed-off-by: Robert Marko <robimarko@gmail.com>

robimarko commented:
Thanks! Rebased on top of main and merged!

wurzerj commented:
Could you please either revert or fix this commit? This currently breaks the NanoPi R4S, the device can't boot anymore. I unfortunately do not have access to the serial logs, but pulling the latest OpenWrt source from Github and reverting this specific commit fixes the problem, so I am pretty sure it is the root cause.

robimarko commented:
Are you still having this issue?

On Sat, 13 Jul 2024 at 01:44, wurzerj ***@***.***> wrote:

> Could you please either revert or fix this commit? This currently breaks
> the NanoPi R4S, the device can't boot anymore. I unfortunately do not have
> access to the serial logs, but pulling the latest OpenWrt source from
> Github and reverting this specific commit fixes the problem, so I am pretty
> sure it is the root cause.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openwrt/openwrt/pull/15815#issuecomment-2226536474>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKMHMAWTNIM4LO5NTFFOY3ZMBS6TAVCNFSM6AAAAABJ7AJRP2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEMRWGUZTMNBXGQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

wurzerj commented:
No, everything is working on the NanoPi R4S.
